cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -213,4 +213,133 @@ spec:
   selector:
     k8s-app: kube-dns
 `
+
+	// CoreDNSDeployment is the CoreDNS Deployment manifest
+	CoreDNSDeployment = `
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: coredns
+  namespace: kube-system
+  labels:
+    k8s-app: coredns
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      k8s-app: coredns
+  template:
+    metadata:
+      labels:
+        k8s-app: coredns
```

Comment:
If you want the kube-dns service to match these Pods, change this to `k8s-app: kube-dns` like `kube-dns` has